src/commands/entities/configure.ts

```diff
@@ -73,18 +73,18 @@ the corresponding property in the entity is not modified.
   ]
 
   static flags = {
-    'anaphora-type': MixFlags.anaphoraTypeFlag,
-    'data-type': MixFlags.dataTypeFlag,
-    dynamic: MixFlags.dynamicFlag,
+    'anaphora-type': MixFlags.anaphoraTypeFlag({isDefaultIgnored: true}),
+    'data-type': MixFlags.dataTypeFlag({isDefaultIgnored: true}),
+    dynamic: MixFlags.dynamicFlag({isDefaultIgnored: true}),
```

Comment:
Although I know we had discussed factory functions for flags in the past, I'm not sure how I feel about this. It seems especially easy for a hapless developer to accidentally include one of these flags without the parens (i.e., as a `function` type object) and wonder why things break. Idiomatically, I think I would prefer leaving the default in the flag and adding the following in `configure.ts`:

``` js
static flags = {
  ...
  'anaphora-type': {
     ...MixFlags.anaphoraTypeFlag,
    default: undefined,
  }
}
```

To me, that spread object with an extra property intuitively means some kind of 'override' is being done, and we know that flags are objects anyhow. I'm curious to hear your thoughts on this as a DevEx issue.

Reply:
I agree. The initial approach fit on a single line and was attractive to me because of that but it does make it more difficult to use. I will change it to what you suggest.

Reply:
done.